### PR TITLE
Flatten state output arrays

### PR DESCRIPTION
**Pull request by vegard-solum-4ss**
## Description
Flatten all state output arrays.

## Checklist
- [x] PR title is descriptive and fit for injection into release notes (see tips below).
- [x] Correct label(s) are used.

PR title tips:
* Use imperative mood.
* Describe the motivation for change, issue that has been solved or what has been improved - not how.
* Examples:
  * Add functionality for Allan variance to smsfusion.simulate
  * Upgrade to support Python 3.10
  * Remove MacOS from CI

### CODE REVIEW

File: src/smsfusion/_ahrs.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def euler(self, degrees: bool = True) -> NDArray[np.float64]:
         Returns
         -------
         euler : numpy.ndarray
-            Euler angles, specifically:  alpha (roll), beta (pitch) and gamma (yaw)
+            Euler angle vector, specifically: alpha (roll), beta (pitch) and gamma (yaw)
```
Comment (ace-e4s) [inline src/smsfusion/_ahrs.py:204]: Technically, Euler angles are not  vectors. Let's not use that term. May be "sequence" instead?

Comment (ace-e4s) [reply src/smsfusion/_ahrs.py:204]: Same for all the occurenes elsewhere.

**Review by ace-e4s — CHANGES_REQUESTED** [review]

File: src/smsfusion/_ahrs.py (hunk 1 above)
Comment (vegard-solum-4ss) [reply src/smsfusion/_ahrs.py:204]: Good point. I can fix that.

Comment (vegard-solum-4ss) [inline src/smsfusion/_ahrs.py:204]:
```suggestion
            Euler angles, specifically: alpha (roll), beta (pitch) and gamma (yaw)
```

File: src/smsfusion/_ins.py [hunk 2]
```diff
@@ -182,7 +182,7 @@ def euler(self, degrees: bool = False) -> NDArray[np.float64]:
         Returns
         -------
         euler : numpy.ndarray
-            Euler angles, specifically:  alpha (roll), beta (pitch) and gamma (yaw)
+            Euler angle vector, specifically: alpha (roll), beta (pitch) and gamma (yaw)
```
Comment (vegard-solum-4ss) [inline src/smsfusion/_ins.py:185]:
```suggestion
            Euler angles, specifically: alpha (roll), beta (pitch) and gamma (yaw)
```

File: src/smsfusion/_ins.py [hunk 3]
```diff
@@ -477,7 +477,7 @@ def euler(self, degrees: bool = False) -> NDArray[np.float64]:
...
```
Comment (vegard-solum-4ss) [inline src/smsfusion/_ins.py:480]:
```suggestion
            Euler angles, specifically: alpha (roll), beta (pitch) and gamma (yaw)
```

File: src/smsfusion/_ins.py [hunk 4]
```diff
@@ -129,10 +129,10 @@ def x(self) -> NDArray[np.float64]:
 
         Returns
         -------
-        x : ndarray
-            State as array of shape (9, 1).
+        x : numpy.ndarray
+            State as array of shape (3,).
```
Comment (ace-e4s) [inline src/smsfusion/_ins.py:133]:
```suggestion
            State as array of shape (9,).
```

**Review by ace-e4s — APPROVED** [review]

**Pull request merged**